index.html

```diff
@@ -1,11 +1,582 @@
+            </div>
+          </div>
+
+          <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/2.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+          <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/3.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+          <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/4.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+          <!-- <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/5.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+          <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/6.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+          <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/7.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+          <div class="col-lg-3 col-md-4 col-xs-6">
+            <div class="supporter-logo">
+              <img src="assets/img/supporters/8.png" class="img-fluid" alt="">
+            </div>
+          </div>
+
+        </div>
+
+        <center>Interested in sponsoring us? Check out our <a href="assets/prospectus.pdf">prospectus</a>, and then send us an <a href="mailto:pathfindersevent@gmail.com?subject=I would like to sponsor MIT Pathfinders">email</a>!</center>
+
+      </div>
+
+    </section> -->
+    
+    <!-- End Sponsors Section -->
+
+  </main><!-- End #main -->
+
+  <!-- ======= Footer ======= -->
+  <footer id="footer">
+    <div class="footer-top">
+      <div class="container">
+        <div class="row">
+
+          <div class="col-lg-3 col-md-6 footer-info">
+            <img src="assets/img/logo.png" alt="TheEvenet" style="height:4em">
+            <p><b>Organizing Committee:</b> Ananya Gangavarapu, Sheshank Shankar, Aryan Mahindra, Mikhail Dmitrienko, Veni Dole, Rohan Iyer, Saurish Srivastava, Tejal Dahake, Aabhas Senapati, Nathan Yap<br>
+            <p><b>Advisory Committee:</b> Poonam Patil, Ramesh Raskar, Abhishek Singh, Khahlil Louisy, Francesco Benedetti, [all MIT ppl]</p>
+          </div>
+
+          <div class="col-lg-3 col-md-6 footer-links">
+            <h4>Useful Links</h4>
+            <ul>
+              <li><i class="fa fa-angle-right"></i> <a href="#">Home</a></li>
+              <li><i class="fa fa-angle-right"></i> <a href="#about">About</a></li>
+              <li><i class="fa fa-angle-right"></i> <a href="#event">Event Information</a></li>
```

Comment:
im not sure if these footer links are updated with the new ssections of the page just confirm that it is

Reply:
yeah it is